src/main/resources/log4j2.xml

```diff
@@ -13,6 +13,11 @@
     <Logger name="org.springframework.web" level="info"/>
     <Logger name="org.apache.catalina.core" level="info"/>
     <Logger name="org.apache.catalina.core.ContainerBase.[Catalina].[localhost]" level="info"/>
+
+    <!-- Leaving debug transactions here but commented as it can be helpful for knowing when begin/commit of transaction is-->
+    <!-- <Logger name="org.hibernate.engine.transaction.internal.TransactionImpl" level="debug"/>
+    <Logger name="org.springframework.transaction" level="debug"/>
+    <Logger name="org.springframework.orm.jpa" level="debug"/> -->
```

Comment:
This was super helpful during debugging the creation and closing of transactions so leaving this breakcrumb in.